package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/dapi-client",
-  "version": "0.9.0-dev.2",
+  "version": "0.9.0-dev.3",
```

Comment:
Require a major version bump (two breaking : sendTransaction and getTransaction)

Reply:
Yeah, we will bump it when releasing `0.9.0`